cmd/ci-op-configs-mirror/main.go

```diff
@@ -73,7 +77,8 @@ func main() {
 		repoInfo.Org = o.toOrg
 
 		rbc.PromotionConfiguration.Disabled = true
-		rbc.PromotionConfiguration.Name = fmt.Sprintf("%s-private", rbc.PromotionConfiguration.Name)
+		rbc.PromotionConfiguration.Name = fmt.Sprintf("%s-priv", rbc.PromotionConfiguration.Name)
```

Comment:
Given we are in a separate namespace now, I *think* we may not even need to change the IS names?

Reply:
I would really like to have this, just in case.  Also, if we keep the same name, this can lead to confusion.

Reply:
/shrug